db/migrate/20180512112934_create_offers.rb

```diff
@@ -0,0 +1,21 @@
+class CreateOffers < ActiveRecord::Migration[5.1]
+  def change
+    create_table :offers do |t|
+      t.decimal :suggested_price
+      t.string :status#, default: 'proposed'
+                #jakie inne statusy? published, bought, paid, sent
+      #tutaj ewentualnie jakaś jeszcze kolumna, jeśli potrzebna do oferty
+      t.timestamps
+
+change_column_default(
+  :offers,
+  :status,
+  from: nil,
+  to: "proposed"
+)
+
+
+
```

Comment:
Jeśli dodajesz default w tej samej migracji, co tworzysz tabelę to nie trzeba dodawać change_column_default, wystarczy przy status: `t.string :status, default: 'proposed'`.  zrób najpierw rollback istniejącej migracji `rake db:rollback`, potem zrób zmiany i jeszcze raz `rake db:migrate` :)

Reply:
Dodatkowo do `belongs_to :product` potrzebujemy klucza w bazie (podpowiedź: add_references w migracji lub t.refrences przy tworzeniu tabeli ;) )